test/index-test.js

```diff
@@ -1,18 +1,82 @@
 const expect = require('expect')
+const createSpy = expect.createSpy;
 
 const fs = require('fs')
 const jsdom = require('mocha-jsdom')
 const path = require('path')
 
 
 describe('index', () => {
-  
+
+  const sourceCode = fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8');
+
   jsdom({
-    src: fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
+    src: sourceCode,
+  })
+
+  describe('`countdown` function', function () {
+    it('should exist', function () {
+      expect(countdown).toExist()
+    })
+
+    it('should have call the given callback function after two seconds', function (done) {
+      this.timeout(3000);
+      const spy = createSpy()
+      countdown(spy)
+
+      expect(spy.calls.length).toEqual(0)
+
+      setTimeout(function () {
+        expect(spy.calls.length).toEqual(1)
+        done();
+      }, 2001)
+    })
   })
   
+  describe('`createMultiplier` function', function () {
+    it('should exist', function () {
+      expect(createMultiplier).toExist()
+    })
+
+    it('should return a function', function () {
+      const doubler = createMultiplier(2)
+      expect(typeof doubler).toEqual('function')
```

Comment:
I don't think so, since we're not actually creating a new instance here. https://github.com/mjackson/expect#tobeastring looks a bit cleaner than my original implementation though!